Kconfig.dependencies

```diff
@@ -50,11 +50,16 @@ config SIDEWALK_CRYPTO
 	imply PSA_WANT_ALG_CTR
 	imply PSA_WANT_ALG_GCM
 	imply PSA_WANT_ALG_CMAC
+	imply PSA_WANT_ALG_ECB_NO_PADDING
+	imply PSA_WANT_ALG_SHA_1
```

Comment:
Did you try build without this config? I'm not sure is it necessary, especially when this algorithm is not safe and [NIST](https://csrc.nist.gov/news/2022/nist-transitioning-away-from-sha-1-for-all-apps) recommends migration to another standards.